src/PowerShellEditorServices/Utility/VersionUtils.cs

```diff
@@ -49,6 +49,11 @@ internal static class VersionUtils
         /// </summary>
         public static bool IsPS7 { get; } = PSVersion.Major == 7;
 
+        /// <summary>
+        /// True if we are running in PowerShell 7, false otherwise.
+        /// </summary>
+        public static bool IsPS7OrGreater { get; } = PSVersion.Major >= 7;
```

Comment:
It feels like this should replace the API above it, no?

Reply:
removed.